locksmith/tsconfig.json

```diff
@@ -56,8 +56,8 @@
     // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
 
     /* Experimental Options */
-    // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
-    // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
```

Comment:
Just some context around this, decorators are currently a Stage 2 draft: https://tc39.github.io/proposal-decorators/

That is, they're not quite standard but other than minor changes, they almost certainly will be. `tsc` compiles them into normal JS, so compatibility should not be a concern.